.github/scripts/version_updater.sh

```diff
@@ -67,7 +68,16 @@ if [[ -z "${role}" ]]; then
 fi
 
 # Get latest version.
-version="$(github_api "repos/${source_repo}/releases/latest" | jq '.tag_name' | tr -d '"v')"
+if [[ "${type}" == "github" ]]
+then
+  version="$(github_api "repos/${source_repo}/releases/latest" | jq '.tag_name' | tr -d '"v')"
+elif [[ "${type}" == "gitlab" ]]
+then
+  version="$(curl --retry 5 --silent --fail "https://gitlab.com/api/v4/projects/${source_repo}/releases"|jq '.[0].tag_name'| tr -d '"v')"
+else
+  echo_red 'Unknown source type. Terminating.'
+  exit 128
+fi
```

Comment:
Not sure if this is ever executed, as you are setting github as the default repo_type

Reply:
You right, but in future something can be changed and that condition can handle such situation.